src/Microsoft.PowerApps.TestEngine/TestInfra/PlaywrightTestInfraFunctions.cs

```diff
@@ -205,8 +205,7 @@ public async Task GoToUrlAsync(string url)
                 Page = await BrowserContext.NewPageAsync();
             }
 
-            // TODO: consider whether to make waiting for network idle state part of the function input
-            var response = await Page.GotoAsync(url, new PageGotoOptions() { WaitUntil = WaitUntilState.NetworkIdle });
```

Comment:
Are there other areas of the code which need this updated or is this the only place we put the "WaitUntil" setting?